Mathlib/Data/LazyList/Basic.lean

```diff
@@ -101,8 +101,7 @@ instance : LawfulMonad LazyList := LawfulMonad.mk'
     | case1 =>
       simp only [Thunk.pure, LazyList.bind, LazyList.traverse, Id.pure_eq]
     | case2 _ _ ih =>
-      simp [LazyList.bind, LazyList.traverse, Seq.seq, Id.map_eq, append, Thunk.pure]
-      rw [← ih]
+      simp [LazyList.bind, LazyList.traverse, Seq.seq, Id.map_eq, append, Thunk.pure, ← ih]
       simp only [Thunk.get, append, singleton, Thunk.pure])
```

Comment:
Do you prefer if I try to combine these?

Reply:
If it's not too much trouble, yes

Reply:
I tried: this leaves additional goals. I have squeezed the first simp instead (the list of lemma stayed the same).

Since the entire file is deprecated, I would prefer to leave it at this level of clean-up.